packages/kit/src/core/adapt/builder.js

```diff
@@ -113,6 +113,10 @@ export function create_builder({ config, build_data, routes, prerendered, log })
 			});
 		},
 
+		getBuildData() {
+			return build_data;
+		},
```

Comment:
I think we can do everything we want in this PR without introducing new API surface area. `writeClient(dest)` returns an array of all the written files, so we could use that in place of `build_data.manifest_data.assets`, and `app_dir` can be accessed as `builder.config.kit.appDir`. 

`BuildData` isn't a public type and it would be better to keep it that way, otherwise it becomes part of the contract and is thus subject to semver

Reply:
Roger that. I'll remove